### DISCUSSION TITLE
Reading Line for Manuscripts

### DISCUSSION THREAD

**Discussion by martingluckman** [openseadragon/openseadragon#2430] [Q&A]
Dear All,

What is the best method to implement a movable reading line for working with manuscripts, we need the ability to enable/disable the line, also to move it's position and also change the colour and thickness.

Many thanks,

Martin

**Comment by pearcetm** [comment]
Have you checked out any of the various annotation plugins available at https://openseadragon.github.io/#plugins? They may not be ideal but could be a starting point at least. I'm working on another plugin for this too, but I can't promise when it might be ready for others to use.
